packages/salesforcedx-vscode-replay-debugger/src/states/frameState.ts

```diff
@@ -0,0 +1,15 @@
+import { SFDC_TRIGGER } from '../constants';
```

Comment:
Add copyright snippet like in the other files.

Reply:
Doh! I'll get right on that.